package.json

```diff
@@ -11,7 +11,37 @@
   "bin": {
     "setup-project": "./setup-script.js"
   },
+  "release": {
+    "branches": [
+      "main"
+    ],
+    "dryRun": true,
```

Comment:
This means that we won't publish anything, we can flip it once the package is ready to be published